pkg/aws/conn.go

```diff
@@ -4,17 +4,18 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/sirupsen/logrus"
+	"os"
 )
 
-func CreateSession(region string) (*session.Session, error) {
+func CreateSession(region string) *session.Session {
 	sess, err := session.NewSession(&aws.Config{
 		Region: aws.String(region)},
 	)
 	if err != nil {
 		logrus.Errorf("Can't connect to AWS: %s", err)
```

Comment:
Maybe you should use `logrus.Fatal` directly since it calls `os.Exists(1)` internally after logging.
https://github.com/sirupsen/logrus/blob/79c5ab66aa2ce7d9ff7b3c437ebc22fcc519a967/entry.go#L336